include/picongpu/particles/bremsstrahlung/Bremsstrahlung.tpp

```diff
@@ -76,13 +77,14 @@ template<
     typename T_ElectronSpecies,
     typename T_PhotonSpecies
 >
-template< typename T_Acc >
-DINLINE
-void Bremsstrahlung<T_IonSpecies, T_ElectronSpecies, T_PhotonSpecies>::init(
-    T_Acc const & acc,
+template<
+    typename T_Acc ,
```

Comment:
The docstrings are missing here as well but the `T_Acc acc` is probably in every kernel, right? So the question arises if it makes sense to document its every occurence. I'd prefer it personally, though.

Reply:
the documentation is in the `hpp` file

Reply:
ah, alright, sorry :)